modules/cnv-deleting-vms.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-delete-vms.adoc
+
+[id="cnv-deleting-vms_{context}"]
+
+= Deleting a virtual machine and PVCs using the CLI
+
+When you delete a virtual machine, the PVC it uses is unbound. If you
+do not plan to bind this PVC to a different VM, delete it, too.
```

Comment:
What happens to the PVC in the other method? If this is a valid concern in the console method, you need to update that one.

Reply:
I'll flag that for QE and follow-up.